### ISSUE TITLE
Addon freezing after finding people, can't accept/leave

### ISSUE THREAD

**Issue by Szpryca** [CosminPOP/LFT#20]
Often addon find successfully people for group but he doesn't make party and show us window "I'm ready/leave". We are not in party and we have freezed window which we can't close and even we can't join new turtle finder until /reload or relog

**Comment by EyeOfTheBeast** [comment]
Confirmed happening to me too.

**Comment by Esk11** [comment]
When this happens there is a LFT message:
[LFT] A member of your group has not accepted the invitation. You are rejoining the queue.

But I do not rejoin the queue, and cannot close the "I'm ready/leave" popup.
I feel like it is related to members joining queue as more than 1 role.

**Comment by fortego** [comment]
Yeah, it might be this. Wasn't able to join being healer/dd. Instant success on chosing only one role.
